auto_trainer/function.yaml

```diff
@@ -46,37 +59,35 @@ spec:
         type: str
         doc: The model's tag to log with
         default: ''
-      - name: label_columns
-        type: Optional[Union[str, List[str]]]
-        doc: The target label(s) of the column(s) in the dataset. for Regression or
-          Classification tasks
-        default: null
       - name: sample_set
         type: DataItem
         doc: A sample set of inputs for the model for logging its stats along the
           model in favour of model monitoring. Can be either a URI or a FeatureVector
         default: null
       - name: test_set
         type: DataItem
-        doc: The test set to train the model with
+        doc: The test set to train the model with.
         default: null
       - name: train_test_split_size
         type: float
-        doc: Should be between 0.0 and 1.0 and represent the proportion of the dataset
-          to include in the test split. The size of the Training set is set to the
-          complement of this value. Default = 0.2
+        doc: if test_set was provided then this argument is ignored. Should be between
+          0.0 and 1.0 and represent the proportion of the dataset to include in the
+          test split. The size of the Training set is set to the complement of this
+          value. Default = 0.2
         default: null
       - name: random_state
         type: int
-        doc: Random state for `train_test_split`
+        doc: 'Relevant only when using train_test_split_size. A random state seed
+          to shuffle the data. For more information, see: https://scikit-learn.org/stable/glossary.html#term-random_state
+          Notice that here we only pass integer values.'
         default: null
       - name: labels
         type: dict
         doc: Labels to log with the model
         default: null
       outputs:
       - default: ''
-      lineno: 110
+      lineno: 116
```

Comment:
How come we don't have `has_kwargs` for the function, now that we added `**kwargs` to the signature?
Same is true for the rest of the entrypoints in the file.

Reply:
It was not available in a stable version, was resolved with one of 1.6.0 rc's